src/plugins/vis_type_vega/public/vega_visualization.ts

```diff
@@ -24,6 +25,14 @@ export const createVegaVisualization = ({
   class VegaVisualization {
     private readonly dataPlugin = getData();
     private vegaView: InstanceType<typeof VegaView> | null = null;
+    private vegaStateRestorer = createVegaStateRestorer({
+      restoreData: false,
+      /**
+       *  exclude Built-in signals
+       *  @see https://vega.github.io/vega/docs/signals/
+       */
+      omitSignals: ['width', 'height', 'padding', 'autosize', 'background'],
```

Comment:
restoring the `Built-in` signals causes issues like: 

https://user-images.githubusercontent.com/20072247/107372060-0dbe3380-6af6-11eb-99c1-f3e7d2e1c77a.mov